### PR TITLE
docs(changelog): add 1.12 changes

### PR DESCRIPTION
**Pull request by GGabriele**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +37,55 @@
...
+
+- Konnect can now be configured via the main `deck` command, while
+  `deck konnect` is now considered deprecated.
+  [#645](https://github.com/Kong/deck/pull/645)
+- Added `--skip-ca-certificates` flag. When present, decK will not attempt to
+  sync CA certificates. This assists with using decK to manage multiple
+  workspaces. CA certificates do not belong to a specific workspace, but can be
+  seen and managed through workspaced endpoints, and decK will attempt to
+  remove them if they are not present in a state file when syncing a workspace,
+  even if they were created for use with configuration in another workspace.
```
Comment (lena-larionova) [inline CHANGELOG.md:54]:
```suggestion
  workspaces. CA certificates do not belong to a specific workspace, but can be
  seen and managed through workspaced endpoints. decK will attempt to
  remove them if they are not present in a state file when syncing a workspace,
  even if they were created for use with configuration in another workspace.
```
Splitting very long sentence for readability.

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +37,55 @@
...
+  [#617](https://github.com/Kong/deck/pull/617)
+- Users can no longer attempt to set default values for fields where a default value
```
Comment (lena-larionova) [inline CHANGELOG.md:56]:
```suggestion
- Users can no longer set default values for fields where a default value
```
Removing "attempt" - they can definitely still attempt it, it'll just fail.

Comment (lena-larionova) [inline CHANGELOG.md:56]:
```suggestion
- Users can no longer set default values for fields where a default value
```
Removing "attempt" - they can definitely still attempt it, it'll just fail.

File: CHANGELOG.md [hunk 3]
```diff
@@ -36,6 +37,55 @@
...
+  does not make sense (such as fields with unique constraints, like `id`), and
+  will print an error indicating the restricted field.
+  [#613](https://github.com/Kong/deck/pull/613)
+- Universal binaries are now available for OS X.
+  [#585](https://github.com/Kong/deck/pull/585)
```
Comment (lena-larionova) [inline CHANGELOG.md:61]: Not an edit but a comment - should we update the decK installation doc to include this info? (that would be here: https://github.com/Kong/docs.konghq.com/blob/deck-1.12/src/deck/installation.md).

Comment (lena-larionova) [inline CHANGELOG.md:61]: Not an edit but a comment - should we update the decK installation doc to include this info? (that would be here: https://github.com/Kong/docs.konghq.com/blob/deck-1.12/src/deck/installation.md).

File: CHANGELOG.md [hunk 4]
```diff
@@ -36,6 +37,55 @@
...
+
+### Fixes
+
+- Fixed unreliable diff outputs due to external dependency (gojsondiff).
+  [#538](https://github.com/Kong/deck/issues/538)
+- De-duplicated `select_tags` in file metadata to avoid erroneous mismatch
+  reports when using `--select-tags` with multiple state files.
+  [#623](https://github.com/Kong/deck/pull/623)
+- Fixed several marshalling and URL construction issues for RBAC endpoint
+  permissions
```
Comment (lena-larionova) [inline CHANGELOG.md:73]:
```suggestion
  permissions.
```

Comment (lena-larionova) [inline CHANGELOG.md:73]:
```suggestion
  permissions.
```

**Comment by hbagdi** [comment]
@rainest Could you please review to make sure that the changelog includes all relevant changes from git history?

**Comment by rainest** [comment]
Nothing that isn't tests, internal refactors, or docs in https://github.com/Kong/deck/pulls?q=is%3Apr+-author%3Aapp%2Fdependabot+is%3Aclosed since 1.11 is missing.

File: CHANGELOG.md (hunk 3 above)
Comment (GGabriele) [reply CHANGELOG.md:61]: Actually that should be transparent to users. We are just packaging what used to be 2 binaries into 1.

**Review by haydenlam — DISMISSED** [review]

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**